state/application.go

```diff
@@ -356,15 +355,13 @@ func (op *DestroyApplicationOperation) destroyOps() ([]txn.Op, error) {
 			op.AddError(err)
 		} else {
 			if n != 0 {
+				// force does not overrule live offers
 				err = errors.Errorf("application is used by %d offer%s", n, plural(n))
-				if !op.Force {
-					return nil, err
-				}
-				op.AddError(err)
+				return nil, errors.Trace(err)
 			}
-			// When we are forcing, regardless of the error and offer count, we want this countOp added.
-			ops = append(ops, countOp)
 		}
+		// When we are forcing, regardless of the error and offer count, we want this countOp added.
+		ops = append(ops, countOp)
```

Comment:
I *think* that this was always in the wrong place. @anastasiamac does it look right here?